include/rsl/algorithm.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include <algorithm>
+#include <vector>
+
+namespace rsl {
+
+/**
+ * @brief Does the collection contain the value
+ * 
+ * @param collection   The input collection
+ * @param value        The value to test for in the collection
+ * 
+ * @return true if the value is found in the collection
+ */
+template<typename T>
+bool does_contain(std::vector<T> const& collection, T const& value) {
```

Comment:
Why `does_contain` instead of `contains`?

Reply:
Because I think the name `contains` is kind of ambiguous about what it does. It was just `contains` in parameter_traits, and I changed it when writing it here, so I'm kind of undecided on this one. I like short names, but I also like clear names they are a function that returns a bool.